wgpu-core/src/pipeline.rs

```diff
@@ -352,6 +352,8 @@ pub enum CreateRenderPipelineError {
         location: wgt::ShaderLocation,
         offset: wgt::BufferAddress,
     },
+    #[error("Two or more attributes were assigned to the same shader lcoation {0}")]
```

Comment:
Unfortunately, missed a typo here (also added a few words while we're at it):

```suggestion
#[error("Two or more vertex attributes were assigned to the same location in the shader: {0}")]
```